service/lib/dinstaller/software/manager.rb

```diff
@@ -0,0 +1,263 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "fileutils"
+require "dinstaller/config"
+require "dinstaller/with_progress"
+require "y2packager/product"
+require "yast2/arch_filter"
+require "dinstaller/software/callbacks"
+
+Yast.import "Package"
+Yast.import "Packages"
+Yast.import "PackageCallbacks"
+Yast.import "Pkg"
+Yast.import "Stage"
+
+# YaST specific code lives under this namespace
```

Comment:
What does this mean?

Reply:
That's an old comment (5718690aa), but it looks wrong. I will get rid of it.